### PR TITLE
[DDW-713] Remove "Decentralization countdown" tada icon and info tab

### PR DESCRIPTION
**Pull request by daniloprates**
This PR removes the "Decentralization countdown" tada icon and info tab.

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test

---

## Test Scenarios

**Scenario 1 - Check "Info" tab is not visible**
```gherkin
Given that Daedalus is synced
When I access to Staking Center
Then I can see that the "Info" tab is no longer visible
```

**Scenario 2 - Check "tada" icon is not visible**
```gherkin
Given that Daedalus is synced
When I check the top bar
Then I can see the "tada" icon is no longer displayed
```

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by gabriela-ponce — APPROVED** [review]

**Review by dmitrii-gaico — APPROVED** [review]

**Review by ManusMcCole — APPROVED** [review]

**Review by miorsufianiohk — APPROVED** [review]
Great work @daniloprates! Tested on 18630 👍

File: source/renderer/app/containers/TopBarContainer.js [hunk 1]
```diff
@@ -28,20 +28,10 @@ export default class TopBarContainer extends Component<Props> {
...
-      epochToFullyDecentralized,
-      isFullyDecentralized,
-    } = networkStatus;
-    const { stakingInfoWasOpen } = staking;
-    const shouldShowDecentralizationTopbarTadaAnimation =
-      isFullyDecentralized && !stakingInfoWasOpen;
-    const shouldShowDecentralizationCountdown =
-      isSynced && !!epochToFullyDecentralized;
+    const { isSynced, syncPercentage, isShelleyActivated } = networkStatus;
+    const hasTadaIconAnimation = false;
```
Comment (danielmain) [inline source/renderer/app/containers/TopBarContainer.js:33]: What about to use 'upper snake' for constants?

File: source/renderer/app/containers/TopBarContainer.js [hunk 2]
```diff
@@ -28,20 +28,10 @@ export default class TopBarContainer extends Component<Props> {
...
+    const hasTadaIcon = false;
```
Comment (danielmain) [inline source/renderer/app/containers/TopBarContainer.js:34]: 'upper snake' here too

**Review by danielmain — CHANGES_REQUESTED** [review]
Overall clean and correct 👌👌👌
just two very small stuff that are very common for naming conventions

**Comment by daniloprates** [comment]
> Overall clean and correct 👌👌👌
> just two very small stuff that are very common for naming conventions

Thanks @danielmain. These variables are temporary and will be replaced in [DDW-712](https://jira.iohk.io/browse/DDW-712), but it's important to have this one ready for the next release. Nevertheless, your suggestions were applied :)

**Review by danielmain — APPROVED** [review]
All ok

**Pull request merged**